app/views/prosecution_cases/show.html.haml

```diff
@@ -1,15 +1,3 @@
 = govuk_page_title(@prosecution_case.prosecution_case_reference, t('generic.case'))
 
-- if v2_hearing_summaries?
-  = render partial: 'prosecution_cases/cd_api/show_v2', locals: { case_summary: @prosecution_case }
-- else
-  .govuk-grid-row
-    .govuk-grid-column-two-thirds
-      = render partial: 'cracked_ineffective_trial', locals: { prosecution_case: @prosecution_case }
-
-      = render partial: 'defendants', locals: { prosecution_case: @prosecution_case }
-
-      = render partial: 'hearing_summaries', locals: { prosecution_case: @prosecution_case }
-
-      - if Rails.configuration.x.display_raw_responses
-        = render partial: 'raw_response', locals: { prosecution_case: @prosecution_case }
\ No newline at end of file
+= render partial: 'prosecution_cases/cd_api/show_v2', locals: { case_summary: @prosecution_case }
```

Comment:
it would be nice to expand the `prosecution_cases/cd_api/show_v2` inside the show view,
since it is the only place it is used.
So, we can remove that partial.

We can the remove the test `prosecution_cases/cd_api/_show_v2.html.haml_spec.rb`, a quick look suggests it does not seems to provide much value.